zanata-war/src/main/webapp/WEB-INF/layout/admin/contact_admin_modal.xhtml

```diff
@@ -7,78 +7,62 @@
   aria-labelledby="myModalLabel" aria-hidden="true">
 
   <script>
-    function cancelEmail() {
-      document.getElementById('#{rich:clientId('subject')}').value = '';
-      document.getElementById('#{rich:clientId('message')}').value = '';
+    $(document).ready(function() {
+      jQuery('#' + jqSelector('#{rich:clientId('contact-admin-subject')}')).
+        val('#{msgs['jsf.message.admin.inquiry.subject']}');
+    });
 
-      jQuery('#{rich:clientId('send')}').removeClass('is-active');
+    function cancelContactAdmin() {
+      jQuery('#' + jqSelector('#{rich:clientId('contact-admin-message')}')).val('');
+
+      jQuery(getContactAdminForm()).find('#send-button').removeClass('is-active');
       return false;
     }
+
+    function getContactAdminForm() {
+      return jQuery('#' + jqSelector('#{rich:clientId('contactAdminForm')}'));
+    }
   </script>
 
   <div class="modal__dialog l--constrain-medium">
     <header class="modal__header">
       <h2 class="modal__title">
-        #{msgs['jsf.ContactAdmin']}
+        #{msgs['jsf.message.admin.inquiry.title']}
       </h2>
       <button type="button"
         class="modal__close button--link"
-        data-dismiss="modal" onclick="cancelEmail();return false;">
+        data-dismiss="modal" onclick="cancelContactAdmin();return false;">
```

Comment:
I think this could be shortened to `onclick="cancelContactAdmin"` since that already returns false. At longest it would be `onclick="return cancelContactAdmin()"`